# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1281409810 25200
#      Mon Aug 09 20:10:10 2010 -0700
# Node ID c318b70bf43766d5cdcf77af30baf7822e225a11
# Parent  dc7e0fbca6f076b337daf6fcb1423c7db886a29a
unimplemented.m: Format output for easier on-screen reading.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,14 @@
+2010-08-08  Rik <octave@nomad.inbox5.com>
+
+	* miscellaneous/unimplemented.m: Add newlines to format output for
+	easier reading.  Format all lines < 80 characters.  Remove 
+	'whitebg' function from unimplemented list.
+
 2010-08-08  Rik <octave@nomad.inbox5.com>
 
 	* @ftp/PKG_ADD: Remove temporary build file from hg version control.
 
 2010-08-09  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__print_parse_opts__.m: Default pstex suffix should be '.ps'.
 
diff --git a/scripts/miscellaneous/unimplemented.m b/scripts/miscellaneous/unimplemented.m
--- a/scripts/miscellaneous/unimplemented.m
+++ b/scripts/miscellaneous/unimplemented.m
@@ -21,45 +21,52 @@
 ## @deftypefn {Function File} {} unimplemented ()
 ## Undocumented internal function.
 ## @end deftypefn
 
 function unimplemented (fcn)
 
   ## Some smarter cases, add more as needed.
   switch (fcn)
-  case "textscan"
-    txt = ["textscan is not implemented. Consider using textread or sscanf."];
+
+  case "quad2d"
+    txt = ["quad2d is not implemented.  Consider using dblquad."];
 
   case "gsvd"
-    txt = ["gsvd is currently not part of Octave. See the linear-algebra package at ",...
-    "@url{http://octave.sf.net/linear-algebra/}."];
+    txt = ["gsvd is not currently part of Octave.  See the linear-algebra",... 
+    "package at @url{http://octave.sf.net/linear-algebra/}."];
 
   case "linprog"
-    txt = ["Octave does not currently provide linprog. Linear programming problems may be solved ",...
-    "using @code{glpk}. Try @code{help glpk} for more info."];
+    txt = ["Octave does not currently provide linprog.  ",...
+    "Linear programming problems may be solved using @code{glpk}.  ",...
+    "Try @code{help glpk} for more info."];
 
   case {"ode113", "ode15i", "ode15s", "ode23", "ode23s", "ode23t", "ode45", "odeget", "odeset"}
-    txt = ["Octave provides lsode for solving differential equations. For more information, ",...
-    "try @code{help lsode}.\n Matlab-compatible ODE functions are provided by the odepkg package. ",...
+    txt = ["Octave provides lsode for solving differential equations.  ",...
+    "For more information try @code{help lsode}.  ",...
+    "Matlab-compatible ODE functions are provided by the odepkg package.  ",...
     "See @url{http://octave.sf.net/odepkg/}."];
+  
+  case "textscan"
+    txt = ["textscan is not implemented.  Consider using textread or sscanf."];
 
   otherwise
     if (ismember (fcn, missing_functions ()))
 
       ## The default case.
 
       txt = sprintf ("The %s function is not yet implemented in Octave.", fcn);
 
     else
       return;
     endif
   endswitch
 
-  txt = [txt, "\nPlease read @url{http://www.octave.org/missing.html} ",...
+  txt = [txt, "\n\n@noindent\nPlease read ",...
+  "@url{http://www.octave.org/missing.html} ",...
   "to find out how you can help with contributing missing functionality."];
 
   warning ("Octave:missing-function",["\n", __makeinfo__(txt)]);
 
 endfunction
 
 function list = missing_functions ()
   persistent list = {
@@ -419,17 +426,16 @@ function list = missing_functions ()
   "waitfor", 
   "warndlg", 
   "waterfall", 
   "wavfinfo", 
   "wavplay", 
   "wavrecord", 
   "web", 
   "whatsnew", 
-  "whitebg", 
   "wk1finfo", 
   "wk1read", 
   "wk1write", 
   "workspace", 
   "xlsfinfo", 
   "xlsread", 
   "xlswrite", 
   "xmlread", 
